### PR TITLE
8264729: Random check-in failing header checks.

### PR DESCRIPTION
**Pull request by JimLaskey**
open/src/java.base/share/native/random/create_ziggurat_tables.c should not be in the sources.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8264729](https://bugs.openjdk.java.net/browse/JDK-8264729): Random check-in failing header checks.

### Reviewers
 * [Erik Joelsson](https://openjdk.java.net/census#erikj) (@erikj79 - **Reviewer**) ⚠️ Review applies to c003d155af53a307fb2b5f66ed0fdaf1bb33ee4c
 * [Brian Burkhalter](https://openjdk.java.net/census#bpb) (@bplb - **Reviewer**) ⚠️ Review applies to c003d155af53a307fb2b5f66ed0fdaf1bb33ee4c

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3343/head:pull/3343` \
`$ git checkout pull/3343`

Update a local copy of the PR: \
`$ git checkout pull/3343` \
`$ git pull https://git.openjdk.java.net/jdk pull/3343/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3343`

View PR using the GUI difftool: \
`$ git pr show -t 3343`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3343.diff">https://git.openjdk.java.net/jdk/pull/3343.diff</a>

</details>

### CODE REVIEW

**Review by erikj79 — APPROVED** [review]

**Review by bplb — APPROVED** [review]

File: test/jdk/java/util/Random/RandomTestBsi1999.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```
Comment (erikj79) [inline test/jdk/java/util/Random/RandomTestBsi1999.java:2]: Shouldn't this be:
2012, 2021,

Comment (JimLaskey) [reply test/jdk/java/util/Random/RandomTestBsi1999.java:2]: The earlier dates were while under development. As far as I know, this is the first time they appear.

**Pull request closed without merging**